robocorp-code/bin/create_env/conda.yaml

```diff
@@ -1,9 +1,9 @@
 channels:
-  - conda-forge
-dependencies:
-  - python=3.8.10
-  - pip=20.1
-  - pip:
-    - robocorp-inspector==0.6.5
-    - robotframework==4.1
+    - conda-forge
```

Comment:
We need to update the base environment with the conda.yaml changes now (as commented on slack, we should add some comments to the `conda.yaml` files so that we don't forget about that...).